services/system/Transact/include/services/system/RTransact.hpp

```diff
@@ -84,9 +84,10 @@ namespace SystemService
    struct TraceClientRow
    {
       psibase::Checksum256         id;
+      uint8_t                      waitFor;
```

Comment:
If the same transaction is submitted by multiple clients, they don't necessarily have the same parameters.

Reply:
For my own information, is there a reasonably common scenario in which the same transaction is submitted by multiple clients? I couldn't really think of why this was even tracked/supported.

Reply:
It's probably not very useful, but if it isn't handled correctly, it can allow exploits.